### ISSUE TITLE
Diagnostic report integrity is not secured

### ISSUE THREAD

**Issue by kelson42** [kiwix/kiwix-android#2258]
Steps to reproduce:
* Ask for a send a diagnostic report
* Choose "Save to Drive" (Google)
* Save it

I get a file with the logs

I should get the whole diagnostic

**Comment by kelson42** [comment]
It looks like it works actually only with sending email... but I don't have an email client on my device configured and this is the case of many users. So the Google Drive option should work (better) or we should at least stop proposing this option (worse).

**Comment by s-ayush2903** [comment]
Should we get device info too stored in the new file like as we have for logs? Also, the same has been addressed in #2153 in this [comment](https://github.com/kiwix/kiwix-android/pull/2153#issuecomment-656556898).

The bug addressed in this issue, only exists in very less number of file-sharing systems, it works fine with other clients

**Comment by s-ayush2903** [comment]
@kelson42 Any ideas about it? (i'm asking as the previous comment got unaddressed), or it's totally fine if it's not a priority :)

**Comment by macgills** [comment]
I guess just append the text we write to the email body also to the log file and we get a quick win

**Comment by s-ayush2903** [comment]
I would like to work on this issue

**Comment by s-ayush2903** [comment]
I was just working with the module and I discovered that as per the current scenario(@`kiwix/develop`), the log files being sent to mail contain a ton of logs now in comparison to its situation and goal while it was being implemented and mentioned in the comment: https://github.com/kiwix/kiwix-android/pull/2153#issuecomment-655491114. I remember you saying:_"To get access to all logs, we'll need special permissions"_, I can't argue/claim these are the complete logs but still so much, that'll make the analysis a pain. [This](https://docs.google.com/document/d/1KqQfqKAg9E_mTBPN39tJ3rzUY9glkHqT_WWxJOxhQ_A/edit?usp=sharing) is the log file(of ~200 pages!!) of which I'm talking about.

Have we leveraged permissions somewhere, I think filtering out the useful logs can be a better option for Proper Diagnosis.
